datafusion/common/Cargo.toml

```diff
@@ -33,13 +33,14 @@ name = "datafusion_common"
 path = "src/lib.rs"
 
 [features]
-avro = ["avro-rs"]
+avro = ["apache-avro"]
 jit = ["cranelift-module"]
 pyarrow = ["pyo3"]
 
 [dependencies]
+cranelift-module = { version = "0.84.0", optional = true }
+apache-avro = { path = "/home/martin/git/apache/avro/lang/rust/avro", features = ["snappy"], optional = true }
```

Comment:
This is temporary until apache-avro 0.14 is released to https://crates.io (hopefully soon!)